torch/csrc/jit/passes/quantization.cpp

```diff
@@ -604,7 +605,11 @@ void FoldQuantNodesIntoInputsOutputs(std::shared_ptr<Graph>& graph) {
 }
 
 void QuantFusion(std::shared_ptr<Graph>& graph) {
-  std::string pattern = R"(
+  // First fuse aten::linear op
```

Comment:
Could we list all the patterns we're rewriting in a comment (in a more compact way then in the actual pattern)? Something like
```
q(linear(dq(a), dq(w), dq(b))) --> to_nchw(fbgemm_linear(prepack(to_nhwc(a)),
                                                         prepack(to_nhwc(w)),
                                                         prepack(to_nhwc(b))))
```